packages/core-components/api-report.md

```diff
@@ -2047,6 +2069,16 @@ export const SidebarSpacer: React_2.ComponentType<
 // @public (undocumented)
 export type SidebarSpacerClassKey = 'root';
 
+// @public
+export const SidebarStateContext: React_2.Context<SidebarStateContextType>;
+
+// @public
+export type SidebarStateContextType = {
```

Comment:
I think it's best if we keep this compatible for now. Let's add the `isMobile` prop to the existing context, but make it optional. While I also wanna rename it I think we should hold off on that too and make it a more isolated refactor.

Reply:
Let's avoid a breaking change here, meaning we keep the `SidebarPinStateContextType`